shopify/wishlist-template/api/actions/startCustomerUpdates.ts

```diff
@@ -57,19 +53,32 @@ export async function run({ params, logger, api, connections }) {
 
   if (!allCustomers.length) return;
 
-  // Remove __typename from the customers
-  allCustomers = allCustomers.map(({ __typename, ...rest }) => rest);
+  // No need to remove __typename as it does not exist on the type
 
   // Start enqueuing email sending jobs
   await api.enqueue(
     api.enqueueSendWishlistEmail,
     {
-      allCustomers: allCustomers.filter((customer) => customer.wishlistCount),
+      allCustomers: allCustomers.filter(
+        (customer) => customer.wishlistCount
+      ) as {
+        id: string;
+        email: string;
+        firstName: string;
+        lastName: string;
+        wishlistCount: number;
+        sendUpdateAt: string | null;
+        updateFrequencyOverride: string;
+        shop: {
+          name: string;
+          defaultUpdateFrequency: string;
+        };
```

Comment:
This looks the same as the type in `sendWishlistEmail` - can we create a shared type?